### PR TITLE
DVRL-51: Install all reporter dependencies as DEV dependencies

### PR DESCRIPTION
**Pull request by JohannesDienst-askui**
Adapted the docs to reflect this requirement.

Also fixed a bug in the example project:

```typescript
await AskUIAllureStepReporter.attachVideo(video);
```

Instead of

```typescript
AskUIAllureStepReporter.attachVideo(video);
```

The latter causes a weird exception because the execution stops but wants to read something from a non-existing temp folder after stopping.

### CODE REVIEW

File: docs/docs/general/05-Integrations/reporting.md [hunk 1]
```diff
@@ -53,11 +53,16 @@ beforeEach(async () => {
...
-  AskUIAllureStepReporter.attachVideo(video);
+  await AskUIAllureStepReporter.attachVideo(video);
 });
 ```
 
 #### Enable the Test Environment `@askui/jest-allure-circus` in `jest.config.ts`
+Install `@askui/jest-allure-circus`` environment:
+
+```bash
+npm install @askui/jest-allure-circus
```
Comment (adi-wan-askui) [inline docs/docs/general/05-Integrations/reporting.md:64]:
```suggestion
npm install --save-dev @askui/jest-allure-circus
```

File: docs/versioned_docs/version-0.12.2/general/05-Integrations/reporting.md [hunk 2]
```diff
@@ -53,11 +53,16 @@ beforeEach(async () => {
...
-  AskUIAllureStepReporter.attachVideo(video);
+  await AskUIAllureStepReporter.attachVideo(video);
 });
 ```
 
 #### Enable the Test Environment `@askui/jest-allure-circus` in `jest.config.ts`
+Install `@askui/jest-allure-circus`` environment:
+
+```bash
+npm install @askui/jest-allure-circus
```
Comment (adi-wan-askui) [inline docs/versioned_docs/version-0.12.2/general/05-Integrations/reporting.md:64]:
```suggestion
npm install --save-dev @askui/jest-allure-circus
```

File: packages/askui-nodejs/src/lib/interactive_cli/create-example-project.ts [hunk 3]
```diff
@@ -103,7 +103,7 @@ export class CreateExampleProject {
       this.helperTemplateConfig['allure_stepreporter_import'] = "import { AskUIAllureStepReporter } from '@askui/askui-reporters';";
       this.helperTemplateConfig['reporter_placeholder'] = 'reporter: new AskUIAllureStepReporter(),';
       this.helperTemplateConfig['allure_stepreporter_attach_video'] = `const video = await aui.readVideoRecording();
-  AskUIAllureStepReporter.attachVideo(video);`;
+  await AskUIAllureStepReporter.attachVideo(video);`;
```
Comment (adi-wan-askui) [inline packages/askui-nodejs/src/lib/interactive_cli/create-example-project.ts:106]: Indentation seems off...

**Review by adi-wan-askui — APPROVED** [review]

File: packages/askui-nodejs/src/lib/interactive_cli/create-example-project.ts (hunk 3 above)
Comment (JohannesDienst-askui) [reply packages/askui-nodejs/src/lib/interactive_cli/create-example-project.ts:106]: I thought the same actually 😆 

But when you render the nunchucks template the code is correctly intended only if you write it this way.

**Pull request merged**